src/Component/Calculator.js

```diff
@@ -1,52 +1,72 @@
+            <button className="p-5" onClick={() => { eventHandler(calculate(obj, '+/-')); }}>+/-</button>
+            <button className="p-5" onClick={() => { eventHandler(calculate(obj, '%')); }}>%</button>
+            <button className="p-5 orange" onClick={() => { eventHandler(calculate(obj, '÷')); }}>÷
+</button>
           </div>
           <div className="calc-row d-flex align-content-stretch">
-            <button className="p-5">7</button>
-            <button className="p-5">8</button>
-            <button className="p-5">9</button>
-            <button className="p-5 orange">x</button>
+            <button className="p-5" onClick={() => { eventHandler(calculate(obj, '7')); }} >7</button>
+            <button className="p-5" onClick={() => { eventHandler(calculate(obj, '8')); }}>8</button>
+            <button className="p-5" onClick={() => { eventHandler(calculate(obj, '9')); }}>9</button>
+            <button className="p-5 orange" onClick={() => { eventHandler(calculate(obj, 'x')); }}>x</button>
           </div>
           <div className="calc-row d-flex align-content-stretch">
-            <button className="p-5">4</button>
-            <button className="p-5">5</button>
-            <button className="p-5">6</button>
-            <button className="p-5 orange">-</button>
+            <button className="p-5" onClick={() => { eventHandler(calculate(obj, '4')); }}>4</button>
+            <button className="p-5" onClick={() => { eventHandler(calculate(obj, '5')); }}>5</button>
+            <button className="p-5" onClick={() => { eventHandler(calculate(obj, '6')); }}>6</button>
+            <button className="p-5 orange" onClick={() => { eventHandler(calculate(obj, '-')); }}>-</button>
           </div>
           <div className="calc-row d-flex align-content-stretch">
-            <button className="p-5">1</button>
-            <button className="p-5">2</button>
-            <button className="p-5">3</button>
+            <button className="p-5" onClick={() => { eventHandler(calculate(obj, '1')); }}>1</button>
+            <button className="p-5" onClick={() => { eventHandler(calculate(obj, '2')); }}>2</button>
+            <button className="p-5" onClick={() => { eventHandler(calculate(obj, '3')); }}>3</button>
             <button className="p-5 orange">+</button>
           </div>
           <div className="calc-row d-flex align-content-stretch">
-            <button className="p-5 button-2">0</button>
-            <button className="p-5">.</button>
-            <button className="p-5 orange">=</button>
+            <button className="p-5 button-2" onClick={() => { eventHandler(calculate(obj, '0')); }}>0</button>
+            <button className="p-5" onClick={() => { eventHandler(calculate(obj, '.')); }}>.</button>
+            <button className="p-5 orange" onClick={() => { eventHandler(calculate(obj, '=')); }}>=</button>
           </div>
         </div>
       </section>
-    );
-  }
-}
```

Comment:
- [ ] The functionality of the calculator is not what it should be. Please work on it as mentioned in the previous milestone review.